backend/regalloc/regalloc_gi.ml

```diff
@@ -1,9 +1,16 @@
 [@@@ocaml.warning "+a-4-30-40-41-42"]
 
+open! Int_replace_polymorphic_compare
 open! Regalloc_utils
 open! Regalloc_gi_utils
 module State = Regalloc_gi_state
 
+module List = struct
```

Comment:
can you please add a CR-someday here saying that we don't need it when we swtich to build with opam's ocaml 5.1?

Reply:
Moved to `Misc` as suggested in
another comment. I reckon we
should have a recurring task to
remove no-longer needed functions
from `Misc`.